acquisitions/src/main/resources/thunderjet/cross-modules/features/cancel-invoice-with-encumbrance.feature

```diff
@@ -156,3 +160,176 @@ Feature: Cancel an invoice with an Encumbrance
     * def transaction = $
     * print 'Encumbrance transaction: ', transaction
     And match $.transactions[0].encumbrance.status == 'Unreleased'
+    And match $.transactions[0].amount == 1
+
+
+  @Positive
+  Scenario: Cancel a paid invoice after closing the order
+    * def fundId = call uuid
+    * def budgetId = call uuid
+    * def orderId = call uuid
+    * def poLineId = call uuid
+    * def invoiceId = call uuid
+    * def invoiceLineId = call uuid
+
+    * print "1. Create finances"
+    * configure headers = headersAdmin
+    * call createFund { id: "#(fundId)" }
```

Comment:
I think you forgot `def v = ...` for these calls

Reply:
Right, I thought we had updated this feature already and I just blindly copy-pasted that. Fixed.